### PR TITLE
Padawan Trial [Jauhar]

### PR DESCRIPTION
**Pull request by majauhar**
## Description
Added my name [Jauhar] for the Padawan trial task.

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

**Comment by rasswanth-s** [comment]
@majauhar  could you try resolving the conflicts with dev branch?

**Comment by majauhar** [comment]
@rasswanth-s I have resolved the conflicts. Could you approve the review to merge it?

**Comment by majauhar** [comment]
@IonesioJunior Could you look into this. I am not sure why there are issues with the merging.

**Review by rasswanth-s — APPROVED** [review]
Great work @majauhar  on your first Padawan PR 🎸

**Comment by majauhar** [comment]
@rasswanth-s That's awesome. Thank you for all the help along the way.

File: packages/syft/tests/trials/skill_test.py [hunk 1]
```diff
@@ -45,8 +46,10 @@ def test_trial_of_skill() -> None:
     assert get_padawans("R2Q4")["amdjed"] == "PASSED"
     assert get_padawans("R2Q4")["Osam"] == "PASSED"
     assert get_padawans("R2Q4")["Mikaela"] == "PASSED"
+    assert get_padawans("R2Q4")["jauhar"]  == "PASSED"
```
Comment (rasswanth-s) [inline packages/syft/tests/trials/skill_test.py:49]: TYPO: First letter is capital in the above dictionary, could you modify it?

Comment (majauhar) [reply packages/syft/tests/trials/skill_test.py]: Done. Please have a look.

**Comment by shubham3121** [comment]
Congratulations @majauhar on your first PR !!! 🎉🚀

**Pull request merged**